packages/editor-ui/src/plugins/connectors/N8nCustomConnector.ts

```diff
@@ -517,7 +517,7 @@ export class N8nConnector extends AbstractConnector {
 	}
 
 	resetTargetEndpoint() {
-		this.overrideTargetEndpoint = null;
+		this.overrideTargetEndpoint = null as unknown as Endpoint;
```

Comment:
seems like we have to fix the types here.

Reply:
Not much I can do. We're extending `AbstractEndpoint` and just using different types. I'm afraid of the side effects of removing all those `null` assignments. Should we try to tackle this separately?

Reply:
Sure maybe we can add a todo or open a story?

Reply:
Created https://linear.app/n8n/issue/N8N-7439/fix-n8ncustomconnector-types